geopandas/io/sql.py

```diff
@@ -93,6 +93,89 @@ def load_geom_text(x):
     return GeoDataFrame(df, crs=crs, geometry=geom_col)
 
 
+def _get_srid_and_geom_from_postgis(name, con, schema=None, geom_name=None):
+    """
+    Get Geometry/Geography column name, SRID, and SQLalchemy type.
+
+    Parameters
+    ----------
+    name : str
+        Name of the target table.
+    con : sqlalchemy.engine.Connection or sqlalchemy.engine.Engine
+        Active connection to the PostGIS database.
+    schema : str, optional
+        Specify the schema. If None, use default schema: 'public'.
+    geom_name : str, optional
```

Comment:
`geom_name` should default to `geom_col` passed to `read_postgis`, since that is always checked.

Reply:
Do you mean changing argument name from `geom_name` to `geom_col`? This function was created only for the `_write_postgis` function and is not used in `read_postgis` (already works correctly).